packer/windows.pkr.hcl

```diff
@@ -0,0 +1,209 @@
+variable "gcp_project" { type = string }
+variable "task_name" { type = string }
+
+variable "build_type" {
+  type = string
+  default = "googlecompute"
+}
+
+variable "docker_repo" {
+  type = string
+  default = ""
+}
+
+variable "docker_server" {
+  type = string
+  default = ""
+}
+
+variable "gcp_password" {
+  type = string
+  default = ""
+}
+
+variable "image_date" {
+  type = string
+  default = ""
+}
+
+variable "prefix" {
+  type = string
+  default = ""
+}
+
+locals {
+  name = "${var.prefix}pg-ci"
+
+  windows_gcp_images = [
+    {
+      name = "${var.task_name}"
+    },
+  ]
+
+  perl_version = "5.26.3.1"
+
+  only_list = {
```

Comment:
I'd just name it 'only', rather than 'only_list', makes the references below a bit shorter, and I don't think the "_list" bit adds much.